build/get_python_test_deps.py

```diff
@@ -38,48 +36,43 @@ def _Download(url, to):
     to_file.write(response.read())
 
 
-def _Unzip(path):
+def _Unzip(path, dir):
   print 'Unzipping %s in %s...' % (path, os.getcwd())
   zip_file = zipfile.ZipFile(path)
   try:
-    zip_file.extractall()
+    zip_file.extractall(dir)
   finally:
     zip_file.close()
 
 
-def _Untar(path):
-  print 'Untarring %s in %s...' % (path, os.getcwd())
-  tar_file = tarfile.open(path, 'r:gz')
+def Install(package):
   try:
-    tar_file.extractall()
-  finally:
-    tar_file.close()
+    print 'Installing python package using pip: ' + package
+    pip.main(['install', '--user' , package])
+  except OSError:
+    print 'Could not install %s due to : %s' % package % OSError
```

Comment:
aha sorry, not sure why the linter did not catch this. And shame on me for not testing it properly.